bundles/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/NetatmoPressureUnit.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (c) 2010-2015, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.netatmo.internal;
+
+import java.math.BigDecimal;
+
+import org.apache.commons.lang.StringUtils;
+
+/**
+ * @author Rob Nielsen
+ * @since 1.8.0
+ * 
+ *         This enum holds all the different pressure units for the Netatmo
+ *         binding, along with conversion methods
+ */
+public enum NetatmoPressureUnit {
+	MBAR("mbar"), INHG("inHg"), MMHG("mmHg");
+
+	public static final NetatmoPressureUnit DEFAULT_PRESSURE_UNIT = NetatmoPressureUnit.MBAR;
+
+	String pressureUnit;
+
+	private NetatmoPressureUnit(String pressureUnit) {
+		this.pressureUnit = pressureUnit;
+	}
+
+	public String getpressureUnit() {
+		return pressureUnit;
+	}
+
+	public static NetatmoPressureUnit fromString(String pressureUnit) {
+		if (!StringUtils.isEmpty(pressureUnit)) {
+			for (NetatmoPressureUnit pressureUnitType : NetatmoPressureUnit
+					.values()) {
+				if (pressureUnitType.toString().equalsIgnoreCase(pressureUnit)) {
+					return pressureUnitType;
+				}
+			}
+		}
+		throw new IllegalArgumentException("Invalid pressureUnit: "
+				+ pressureUnit);
+	}
+	
+	public BigDecimal convertPressure(BigDecimal value) {
+		if (this == DEFAULT_PRESSURE_UNIT) {
+			return value;
+		} else if (this == INHG) {
+			return value.multiply(new BigDecimal(0.02952998751));
```

Comment:
If the BigDecimal multiplier values were static final class variables, you wouldn't have to "new" them with each conversion, which might be more efficient.